pontoon/base/static/js/fluent_interface.js

```diff
@@ -69,7 +69,7 @@ var Pontoon = (function (my) {
                   '<label class="id built-in" for="ftl-id-' + this + '">' +
                     '<span>' + this + ' (e.g. </span><span class="stress">' + Pontoon.locale.examples[pluralForm] + '</span>)<sub class="fa fa-remove remove" title="Remove"></sub>' +
                   '</label>' +
-                  '<input class="value" id="ftl-id-' + this + '" type="text" value="' + value + '">' +
+                  '<input class="value" id="ftl-id-' + this + '" type="text" value="' + value + '" dir="' + Pontoon.locale.direction + '" data-script="' + Pontoon.locale.script + '"  lang="' + Pontoon.locale.code + '">' +
```

Comment:
Double space in front of `  lang=`.

Reply:
Resolved, generation of this element is now contained in a function